src/target/riscv/riscv.h

```diff
@@ -124,8 +124,10 @@ typedef struct {
 	/* record the tinfo of each trigger */
 	unsigned int trigger_tinfo[RISCV_MAX_TRIGGERS];
 
-	/* For each physical trigger, contains -1 if the hwbp is available, or the
-	 * unique_id of the breakpoint/watchpoint that is using it.
+	/* For each physical trigger contains:
+	 * -1: the hwbp is available
+	 * -4: The trigger is used by the itrigger command
```

Comment:
Now there is also the `-5` value for etriggers.

```suggestion
	 * -4: The trigger is used by the itrigger command
	 * -5: The trigger is used by the etrigger command	 
```

Reply:
Done.